docs/admin/cluster-management.md

```diff
@@ -22,41 +22,33 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Management
 
-This doc is in progress.
-
 ## Upgrading a cluster
 
-The `cluster/kube-push.sh` script will do a rudimentary update; it is a 1.0 roadmap item to have a robust live cluster update system.
-
-## Updgrading to a different API version
-
-There is a sequence of steps to upgrade to a new API version.
-
-1. Turn on the new api version
-2. Upgrade the cluster's storage to use the new version.
-3. Upgrade all config files. Identify users of the old api version endpoints.
-4. Update existing objects in the storage to new version by running cluster/update-storage-objects.sh
-3. Turn off the old version.
-
-### Turn on or off an API version for your cluster
-
-Specific API versions can be turned on or off by passing --runtime-config=api/<version> flag while bringing up the server. For example: to turn off v1 API, pass --runtime-config=api/v1=false.
-runtime-config also supports 2 special keys: api/all and api/legacy to control all and legacy APIs respectively. For example, for turning off all api versions except v1, pass --runtime-config=api/all=false,api/v1=true.
-
-### Switching your cluster's storage API version
+The current state of cluster upgrades is provider dependent.
```

Comment:
Should you mention in this section that for GKE and GCE master upgrades are done by provisioning a new VM hooked up to the old etcd PD, while kube-push.sh just upgrades the binaries in-place? (You cold also explain nodes -- MIG vs. in-place binary upgrade)